src/components/AddressSearch/index.tsx

```diff
@@ -224,6 +233,30 @@ function AddressSearch(
 
         onPress?.(values);
     };
+    const selectFocusedOption = () => {
```

Comment:
Let's memoize this with `useCallback` so `useKeyboardShortcut` doesn't add and remove an event listener on every render.